setup.py

```diff
@@ -256,7 +257,9 @@ def run(self):
 
 extras["serving"] = deps_list("pydantic", "uvicorn", "fastapi", "starlette")
 extras["audio"] = deps_list("librosa", "pyctcdecode", "phonemizer")
-extras["speech"] = deps_list("torchaudio") + extras["audio"]  # `pip install ".[speech]"` is deprecated and `pip install ".[torch-speech]"` should be used instead
+extras["speech"] = (
+    deps_list("torchaudio") + extras["audio"]
+)  # `pip install ".[speech]"` is deprecated and `pip install ".[torch-speech]"` should be used instead
```

Comment:
Please put the comment above to avoid this ugly formatting.

Reply:
I didn't touch this actually is this file blacked ? (I could revert too this part) (Everything is blacked automatically as a pre-commit so maybe that explains it)

Reply:
Your precommit executes something different than `make style` then, because `make style` only runs on the following dirs: examples tests src utils.

Reply:
Yes I black all files that ends with `.py` (I took the default `pre-commit` setup for python).

Overall I would say the result for this file is actually better, but happy to revert if anyone disagrees.

Reply:
Something like that. https://ljvmiranda921.github.io/notebook/2018/06/21/precommits-using-black-and-flake8/

Reply:
I'm good with the result, as long as the comment is moved.

Reply:
It's done ! Should be good.